### ISSUE TITLE
Portfolio

### ISSUE THREAD

**Issue by StantonR16** [LilgaE/csPortfolio#1]
Your portfolio is pretty good, but I would use some formatting like this:

<details><summary><strong>Click me</strong></summary>

Stuff

</details>

Also complete your questions.

What does your code do?

**Comment by kantab** [comment]
I disagree with Read, I think the layout of it is nice and does not have to use dropdowns.
